src/pages/[...slug].tsx

```diff
@@ -333,6 +326,20 @@ export async function getStaticPaths(
     }
   }
 
+  if (process.env.CHERRY_PICKED_PATHS) {
+    log('CHERRY_PICKED_PATHS is set, returning only cherry-picked paths')
+    return {
+      paths: [
+        {
+          params: {
+            slug: ['alexandria-va-vet-center', 'locations'],
+          },
+        },
+      ],
+      fallback: 'blocking',
+    }
+  }
+
```

Comment:
This is what I was using to compare apples to apples--static build to static build. I didn't want to build _alllll_ the pages, so I did this to build only the one I was looking at.

We can 100% remove it, or we can make this nicer (either reading the env var, or reading paths from a file if the env var is truthy). I'm not picky either way.